File: 03-easy/file_size/file.size.java
// Reuben Thorpe (2016), CodeEval [File Size v1.0]
import java.io.File;

public class Main {
    public static void main (String []args) {
        File file = new File(args[0]);
        if (file.exists()) {
            System.out.println((int)file.length());
        }
        else {
            System.out.println("Could not find file : " + args[0]);
        }
    }
}

File: 03-easy/file_size/file.size.c
// Reuben Thorpe (2016), CodeEval [File Size v1.4]
#include <stdio.h>

int main(int argc, const char * argv[]) {
    FILE *file = fopen(argv[1], "r");
    fseek(file, 0L, SEEK_END);
    printf("%i", ftell(file));
    return(0);
}
